apps/airdrop/src/App/Common/AcceptTermsCheckbox/AcceptTermsCheckbox.tsx

```diff
@@ -1,27 +1,70 @@
 import { Checkbox } from "@namada/components";
-import { TOSToggle } from "./AcceptTermsCheckbox.components";
+import { TOSFullTerms, TOSToggle } from "./AcceptTermsCheckbox.components";
 
 type AcceptTermsCheckboxProps = {
   onChange: () => void;
   checked: boolean;
   disabled?: boolean;
+  fullVersion?: boolean;
 };
 
 export const AcceptTermsCheckbox = ({
   disabled = false,
+  fullVersion = false,
   checked,
   onChange,
 }: AcceptTermsCheckboxProps): JSX.Element => {
   return (
     <TOSToggle disabled={disabled}>
```

Comment:
@pedrorezende 
One thing've noticed is that before when toggle was disabled it had different color:
<img width="590" alt="image" src="https://github.com/anoma/namada-interface/assets/11858303/395486af-1597-4fe0-8c6a-273ec8cdb55a">
now it's always bright yellow:
<img width="577" alt="image" src="https://github.com/anoma/namada-interface/assets/11858303/05ef149b-a919-4d6d-810f-f9efcce059b6">
It might be by design though,  so the text is easier to read.